.gitignore

```diff
@@ -1,8 +1,15 @@
 # Visual Studio
 .vs/
+.DS_Store
+Notify.userprefs
+NotifyTests/.DS_Store
```

Comment:
Don't think you need to add another entry here for `.DS_Store` as should be covered by `.DS_Store`